target/linux/lantiq/patches-4.19/0720-NET-reactivate-ndo_do_ioctl-fallback.patch

```diff
@@ -0,0 +1,34 @@
+From 6a7056cfd86810dad031309cb301443cbca2188b Mon Sep 17 00:00:00 2001
+From: Quallenauge <Hamsi2k@freenet.de>
+Date: Mon, 21 May 2018 23:55:01 -0100
+Subject: [PATCH] Revert "wireless: wext: remove ndo_do_ioctl fallback"
+X-Patchwork-Bot: notify
+
+This reverts commit 8bfb3676606454ffec836f56c5dc3e69dfc0956a.
+
+The ralink inic driver currently uses this for controlling the network
+adapters. In future versions the driver has to be modernized to comply
+with future kernel drivers.
+
```

Comment:
I am hoping we will not need any device specific drivers. Rather only use mainline ones.

Which interface is this about, can we maybe call it optional to start with?

I guess https://github.com/Quallenauge/ralink_inic via https://github.com/OpenWrt-Repository/openwrt-feeds-easybox904-driver/blob/master/kernel/ralink_bin/Makefile which according to @dangowrt should already work and does not need to be added.